Mathlib/RingTheory/Polynomial/SmallDegreeVieta.lean

```diff
@@ -0,0 +1,55 @@
+/-
+Copyright (c) 2025 Qinchuan Zhang. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Qinchuan Zhang
+-/
+import Mathlib.Tactic.FieldSimp
+import Mathlib.Tactic.LinearCombination
+import Mathlib.RingTheory.Polynomial.Vieta
+
+/-!
+# Vieta's Formula for polynomial of small degrees.
+-/
+
+namespace Multiset
+
+lemma esymm_one_of_pair {R : Type*} [CommSemiring R] (x1 x2 : R) :
+    esymm {x1, x2} 1 = x2 + x1 := by
+  simp [esymm, powersetCard_one]
+
+lemma esymm_two_of_pair {R : Type*} [CommSemiring R] (x1 x2 : R) :
+    esymm {x1, x2} 2 = x1 * x2 := by
+  simp [esymm, powersetCard_one]
```

Comment:
These lemmas are misnamed; `of` means "implies" in the naming convention. They should be `esymm_pair_one` and `esymm_pair_two` and they should in another file, for example the file where `esymm` is defined.